packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -1136,6 +1136,16 @@ type MessagesType = {
   'modals.simplebuy.name_on_card': 'Name on Card'
   'modals.simplebuy.ordersummary': 'Buy Order Summary'
   'modals.simplebuy.refresh': 'Refresh'
+  'modals.simplebuy.rejected.blocked_by_laws': 'Blocked by Local Laws'
+  'modals.simplebuy.rejected.blocked_by_laws_description': 'At Blockchain.com, we strive to adhere to any and all local laws. Based on your location, we cannot allow the buying or selling digital currencies at this time.'
```

Comment:
buying or selling `of` digital currencies

Reply:
I see, I copy/pasta it from figma :)